packages/app/src/types.ts

```diff
@@ -67,6 +67,26 @@ export type AlertChannelType = 'webhook';
 
 export type Alert = z.infer<typeof AlertSchema>;
 
+export enum AlertState {
+  ALERT = 'ALERT',
+  DISABLED = 'DISABLED',
+  INSUFFICIENT_DATA = 'INSUFFICIENT_DATA',
+  OK = 'OK',
+}
+
+export type AlertHistory = {
+  counts: number;
+  createdAt: string;
+  lastValues: { startTime: string; count: number }[];
+  state: AlertState;
+};
```

Comment:
should be able to import these from `common-utils` pkg

Reply:
updated